docs: blocks limits (#502)
tinacms/tinacms.org: content/docs/inline-editing/inline-blocks.md

  itemProps?: {
    [key: string]: any
  }
  min?: number
    max?: number
}
```

| Key | Purpose |
| --- | ---: |
| `name` | The path to the **source data** for the blocks. |
| `blocks` | An object composed of individual [Blocks](/docs/inline-editing/inline-blocks#creating-a-block). |
| `className` | To set styles directly on the input or extend via [styled components](/docs/inline-editing#extending-inline-field-styles). |
| `direction` | Sets the orientation of the drag direction and `AddBlock` button position. |
| `itemProps` | An object that passes additional props to every block child element. |

> | `min` | Controls the minimum number of blocks. Once reached, blocks won't be able to be removed. |
> | `max` | Controls the maximum number of blocks allowed. Once reached, blocks won't be able to be added. |

Reviewer: ```suggestion
| `min` | Controls the minimum number of blocks. Once reached, blocks won't be able to be removed. _(Optional)_ |
| `max` | Controls the maximum number of blocks allowed. Once reached, blocks won't be able to be added. _(Optional)_ |
```

  itemProps?: {
    [key: string]: any
  }

>   limits?: {

Author: ```suggestion
```

>   	min?: number

Author: ```suggestion
  min?: number
```

    max?: number

>   }

Author: ```suggestion
```

}
```

| Key | Purpose |
| --- | ---: |
| `name` | The path to the **source data** for the blocks. |
| `blocks` | An object composed of individual [Blocks](/docs/inline-editing/inline-blocks#creating-a-block). |
| `className` | To set styles directly on the input or extend via [styled components](/docs/inline-editing#extending-inline-field-styles). |
| `direction` | Sets the orientation of the drag direction and `AddBlock` button position. |
| `itemProps` | An object that passes additional props to every block child element. |

> | `limits` | Controls how many blocks can be added / removed. |

Author: ```suggestion
| `min` | Controls the minimum number of blocks. Once reached, blocks won't be able to be removed. |
```

> 

Author: ```suggestion
| `max` | Controls the maximum number of blocks allowed. Once reached, blocks won't be able to be added. |

```
